src/components/templates/SubProjectSection/SubProjectSection.tsx

```diff
@@ -41,25 +42,36 @@ interface LeftComponentProps {
   ext_id: string
   price?: string
   deadline_at?: string
+  mode?: ProjectDetailModes
 }
 
-const LeftComponent: FC<LeftComponentProps> = ({
+export const LeftComponent: FC<LeftComponentProps> = ({
```

Comment:
**[non-blocker]**

Not too important, but usually we only keep these helper components inside the parent component file, if they are not used anywhere else. Once you start using it somewhere else, it should be moved to it's own separate folder as a component (and possibly renamed in this case, since LeftComponent is very hard to understand)